opensrp-tasking/src/main/java/org/smartregister/tasking/adapter/TaskRegisterAdapter.java

```diff
@@ -115,7 +115,7 @@ public void onBindViewHolder(@NonNull TaskRegisterViewHolder viewHolder, int pos
             viewHolder.hideHouseNumber();
         }*/
 
-        TaskingLibrary.getInstance().getTaskingLibraryConfiguration().onTaskRegisterBindViewHolder(context, viewHolder, registerActionHandler, taskDetails.get(position), position);
+        TaskingLibrary.getInstance().getTaskingLibraryConfiguration().onTaskRegisterBindViewHolder(context, (TaskRegisterViewHolder) viewHolder, registerActionHandler, taskDetails.get(position), position);
```

Comment:
Casting this causes a crash on my branch and make it coupled to the TaskRegisterViewHolder

Reply:
You can also change the param type on `onTaskRegisterBindViewHolder` to `RecyclerView.ViewHolder`

Reply:
cool